### ISSUE TITLE
timesnet has different val and test performances on my dataset

### ISSUE THREAD

**Issue by dongxinyu1030** [thuml/Time-Series-Library#107]
I compared the performance of TimesNet, Dlinear, and Informer on my dataset. I observed that TimesNet's validation and test performances are inconsistent, with different decreasing trends in loss. However, both Dlinear and Informer exhibit similar trends in loss.
When validate, the batch size = 128, when test, the batch size = 1.
Could anyone help with this issue?

Here are the loss curves of three models:
<img width="912" alt="截屏2023-05-29 16 53 36" src="https://github.com/thuml/Time-Series-Library/assets/57408822/a7ac50f9-943a-447f-993d-1f8d24f09925">

Settings of training:
features S
seq_len = 150
label_len = 75
pred_len = 75
e_layers = 3
d_layers = 1
factor = 3
enc_in = 1
dec_in = 1
c_out = 1
d_model = 64
d_ff = 256
top_k = 3
batch_size 128

**Comment by wuhaixu2016** [comment]
Hi, thanks for your question.
If the enc_in and c_out are 1, you can decrease d_model and d_ff. Based on my experiences, d_model=d_ff=32 can bring better performance.

**Comment by dongxinyu1030** [comment]
Anyone who has similar issues could try to reduce the complexity of the network. I tried what @wuhaixu2016 mentioned, and now the test loss reduces smoothly along with the validation loss. 
Here are the new loss curves for each setting:
<img width="911" alt="截屏2023-05-30 12 05 29" src="https://github.com/thuml/Time-Series-Library/assets/57408822/df1db66c-4891-4969-aa62-ef04673c6492">
